api/users.go

```diff
@@ -86,6 +87,31 @@ func (api *API) ListUsersHandler(ctx context.Context, w http.ResponseWriter, req
 	return models.NewSuccessResponse(jsonResponse, http.StatusOK, nil), nil
 }
 
+//GetUserHandler lists the users in the user pool
+func (api *API) GetUserHandler(ctx context.Context, w http.ResponseWriter, req *http.Request) (*models.SuccessResponse, *models.ErrorResponse) {
+	vars := mux.Vars(req)
+	user := models.UserParams{ID: vars["id"]}
+	getUserInput := user.BuildAdminGetUserRequest(api.UserPoolId)
```

Comment:
The variables names `getUserInput` and `getUserResp` can be `userInput` and `userResp` respectively. Functions and methods should be named with a _verb_ while variables should be named with a _noun_.

https://softwareengineering.stackexchange.com/questions/33532/naming-conventions-for-variables

Reply:
I've updated the variable names, thanks for the guidance on naming conventions.